src/Services/AlignmentSentinelService.php

```
@@ -0,0 +1,283 @@
+     * story that has been blocked at least $maxDays days. Severity escalates to
+     * 'critical' at 10+ days.
+     *
+     * @param int $projectId Project to scan (must belong to $orgId — caller's responsibility)
+     * @param int $orgId     Organisation ID (passed to DriftAlert lookup for tenant isolation)
+     * @param int $maxDays   Minimum days blocked before an alert is raised (default 5)
+     * @return array         List of created alert summaries: [alert_id, story_id, days_blocked]
+     */
+    public function checkBlockedDuration(int $projectId, int $orgId, int $maxDays = 5): array
+    {
+        $stories = UserStory::findByProjectId($this->db, $projectId);
+        $now     = new \DateTimeImmutable();
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '151', column '24'). (undefined)

(MissingImport)